src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -181,6 +182,7 @@ def _evaluate_operator(self, condition_type: Condition, value: T) -> DataConditi
 
         return result
 
+    @scopedstats.timer()
```

Comment:
fyi - this will also record for detectors. should we also instrument those methods?